### PR TITLE
Add lint

### PR DESCRIPTION
**Pull request by ostatni5**

### CODE REVIEW

File: examples/dashboard/src/components/CameraTest.tsx [hunk 1]
```diff
@@ -1,16 +1,16 @@
-import React, { useEffect, useMemo, useState } from "react";
+import { useEffect, useState } from "react";
 import VideoPlayer from "./VideoPlayer";
 import { useLocalStorageState, useLocalStorageStateString } from "./LogSelector";
 import { JsonComponent } from "./JsonComponent";
-import { enumerateDevices, EnumerateDevices, useUserMediaById } from "../../../../src/navigator";
+import { enumerateDevices, EnumerateDevices, useUserMediaById } from "@jellyfish-dev/jellyfish-react-client/navigator";
 
-type IdToStream = Record<string, MediaStream>;
+// type IdToStream = Record<string, MediaStream>;
```
Comment (kamil-stasiak) [inline examples/dashboard/src/components/CameraTest.tsx:7]:
```suggestion
```

File: .eslintignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+node_modules
+dist
```
Comment (kamil-stasiak) [inline .eslintignore:2]:
```suggestion
node_modules
dist

```

File: examples/dashboard/.eslintignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+node_modules
+dist
```
Comment (kamil-stasiak) [inline examples/dashboard/.eslintignore:2]:
```suggestion
node_modules
dist

```

File: src/externalState/index.ts [hunk 4]
```diff
@@ -1 +1 @@
-export { createNoContextMembraneClient } from "./createNoContextMembraneClient";
+export { createNoContextMembraneClient } from "./createNoContextMembraneClient";
```
Comment (kamil-stasiak) [inline src/externalState/index.ts:1]:
```suggestion
export { createNoContextMembraneClient } from "./createNoContextMembraneClient";

```

File: src/jellyfish/index.ts [hunk 5]
```diff
@@ -0,0 +1,2 @@
+export * from './JellyfishClient'
+export * from './addLogging'
```
Comment (kamil-stasiak) [inline src/jellyfish/index.ts:2]:
```suggestion
export * from './JellyfishClient'
export * from './addLogging'

```

File: src/stateMappers.tsx [hunk 6]
```diff
@@ -2,30 +2,30 @@ import type { Peer as WebRtcPeer, SimulcastConfig, TrackContext } from "@jellyfi
 import type { Peer, PeerId, State, Track, TrackId } from "./state.types";
 
 export const onSocketOpen =
-  <PeerMetadata, TrackMetadata>(metadata?: any) =>
+  <PeerMetadata, TrackMetadata>(_metadata?: unknown) =>
```
Comment (kamil-stasiak) [inline src/stateMappers.tsx:5]: investigate metadata type

File: tsconfig.json [hunk 7]
```diff
@@ -5,11 +5,11 @@
     "declarationMap": true,
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
-    "module": "commonjs",
+    "module": "ESNext",
```
Comment (kamil-stasiak) [inline tsconfig.json:8]: investigate Motule resolution

**Review by kamil-stasiak — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 7 above)
Comment (ostatni5) [reply tsconfig.json:8]: [Browsers support `ES` modules. ](https://www.tsmean.com/articles/learn-typescript/typescript-module-compiler-option/)

**Pull request merged**